geowebcache/core/src/test/java/org/geowebcache/grid/BoundingBoxTest.java

```diff
@@ -41,26 +43,28 @@ public void testBBOXScale() throws Exception {
         assert (bbox.equals(copy));
     }
 
+    @Test
     public void testIntersection() throws Exception {
         BoundingBox bb1 = new BoundingBox(0, 0, 10, 10);
         BoundingBox bb2 = new BoundingBox(5, 5, 20, 20);
 
         BoundingBox intersection = BoundingBox.intersection(bb1, bb2);
-        assertNotNull(intersection);
-        assertEquals(5D, intersection.getWidth());
-        assertEquals(5D, intersection.getHeight());
-        assertTrue(intersection.isSane());
+        Assert.assertNotNull(intersection);
+        Assert.assertEquals(5D, intersection.getWidth(), 0d);
+        Assert.assertEquals(5D, intersection.getHeight(), 0d);
```

Comment:
Did a double take at `5D` vs `0d`, but it is not case sensitive ... I just have never see `D` before.

Reply:
Both are valid, I prefer the "d" form. The capital one was there already I believe.